src/components/App.js

```diff
@@ -0,0 +1,17 @@
+import '../style.css';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+function App() {
```

Comment:
## Enforce the use of `ES6` syntax
- [ ] Kindly change this function declaration into an arrow function.